libgcc/config.host

```diff
@@ -397,10 +397,10 @@ case ${host} in
   *-*-solaris2*)
   enable_execute_stack=enable-execute-stack-mprotect.c
   ;;
-i[34567]86-*-mingw* | x86_64-*-mingw*)
+i[34567]86-*-mingw* | x86_64-*-mingw* | aarch64-*-mingw*)
```

Comment:
I think it might be replaced with *-mingw*

Reply:
Changed.